actors/builtin/multisig/multisig_state.go

```diff
@@ -33,11 +34,21 @@ func (st *State) AmountLocked(elapsedEpoch abi.ChainEpoch) abi.TokenAmount {
 	return big.Mul(unitLocked, big.Sub(big.NewInt(int64(st.UnlockDuration)), big.NewInt(int64(elapsedEpoch))))
 }
 
-func (st *State) isSigner(party address.Address) bool {
-	for _, ap := range st.Signers {
+func (st *State) isSigner(rt vmr.Runtime, party address.Address) bool {
```

Comment:
It is a rule in actor code that the state object must never use the runtime object, since this makes such methods impossible to call from outside actor code. Application logic like resolving addresses belongs in the actor itself.

(There are a few lingering violations of this rule)

Reply:
@arajasek i'd just make this a method on the actor then, pass in the state as well, and move this function to the actor file.